idpo/.htaccess

```diff
@@ -0,0 +1,52 @@
+Options +FollowSymLinks
+RewriteEngine on
+
+# Turn off MultiViews
+Options -MultiViews
+
+AddType text/turtle .ttl
+AddType application/rdf+xml .rdf
+AddType application/ld+json .jsonld
+
+# Rewrite rule to serve HTML content from the vocabulary URI if requested
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://icdd.vm.rub.de/ontology/idpo/ [R=302,NE,L]
+
+# In case of accept header <text/turtle>
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+RewriteRule ^$ https://icdd.vm.rub.de/ontology/idpo/idpo.ttl [R=303,NE,L]
+
+# In case of accept header <application/rdf+xml>
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^$ https://icdd.vm.rub.de/ontology/idpo/idpo.rdf [R=302,NE,L]
+
+# In case of accept header <application/ld+json>
+RewriteCond %{HTTP_ACCEPT} application/ld+json
+RewriteRule ^$ https://icdd.vm.rub.de/ontology/idpo/idpo.jsonld [R=302,NE,L]
+
+# If suffix ttl, redirect to turtle version
+RewriteRule ^idpo.ttl$ https://icdd.vm.rub.de/ontology/idpo/idpo.ttl [R=302,NE,L]
```

Comment:
This will be https://w3id.org/idpo/idpo.ttl Is this what you intend?

Reply:
Right, its intented to serve the turtle file either by requesting using the <text/turtle> header on the base path or specifying the ttl file directly, e.g for weblinks to that file